js/myAPI.js

```diff
@@ -0,0 +1,15 @@
+const myURL = 'http://127.0.0.1:5500/static/';
```

Comment:
have to be changed when we go live, but works in vs code if you use live server...

Reply:
You don't need the whole path, this is enough:
`await fetch("./static/database.json")`

Reply:
Indeed you are right, "fetch("static/database.json")" works. When I tried to do something similar I got lots of errors, but this works, thanks!